shared/domain/build.gradle.kts

```diff
@@ -1,14 +1,23 @@
 plugins {
     id("ivy.feature")
+    id("ivy.integration.testing")
+    id("ivy.room")
 }
 
 android {
-    namespace = "com.ivy.core"
+    namespace = "com.ivy.domain"
 }
 
 dependencies {
     implementation(projects.shared.base)
     implementation(projects.shared.data)
 
-    implementation(libs.datastore) // TODO: Will be removed
+    implementation(libs.datastore)
+    implementation(libs.bundles.ktor)
+
+    val mockkVersion = "1.13.10"
+    androidTestImplementation(libs.bundles.integration.testing)
+    androidTestImplementation(projects.shared.testing)
+    androidTestImplementation("io.mockk:mockk-android:$mockkVersion")
```

Comment:
Use the version catalog - all dependencies must be defined there - `libs.versions.toml`